pkg/drivers/pgsql/pgsql.go

```diff
@@ -135,16 +135,17 @@ func createDBIfNotExist(dataSourceName string) error {
 
 	dbName := strings.SplitN(u.Path, "/", 2)[1]
 	u.Path = "/postgres"
+	var exists bool
 	db, err := sql.Open("pgx", u.String())
 	if err != nil {
-		return err
-	}
-	defer db.Close()
+		logrus.Warnf("failed to check existence of database %s; unable to connect to postgres database: %v", dbName, err)
+	} else {
+		defer db.Close()
 
-	var exists bool
-	err = db.QueryRow("SELECT 1 FROM pg_database WHERE datname = $1", dbName).Scan(&exists)
-	if err != nil && err != sql.ErrNoRows {
-		return err
+		err = db.QueryRow("SELECT 1 FROM pg_database WHERE datname = $1", dbName).Scan(&exists)
```

Comment:
can you scan an untyped int into a bool?

Reply:
This is just an indentation change, so no change really. But yes, you can. From the docs for `Rows.Scan`:

> For scanning into *bool, the source may be true, false, 1, 0, or string inputs parseable by strconv.ParseBool.